backend/app/views/spree/admin/orders/_risk_analysis.html.erb

```diff
@@ -1,50 +1,26 @@
-<fieldset class="no-border-bottom" id="risk_analysis">
-  <legend><%= "#{t('spree.risk_analysis')}: #{t('spree.not') unless @order.is_risky?} #{t('spree.risky')}" %></legend>
+<details id="risk_analysis">
+  <summary><%= t('spree.risk_analysis') %>: <%= t('spree.risky') %></summary>
+
   <table>
     <thead>
+      <th><%= t('spree.payment') %></th>
       <th><%= t('spree.risk') %></th>
       <th><%= t('spree.status') %></th>
     </thead>
-    <tbody id="risk-analysis" data-hook="order_details_adjustments" class="with-border">
-      <tr class="">
-        <td><strong>
-          <%= t('spree.failed_payment_attempts') %>:
-        </strong></td>
-        <td>
-          <span class="pill pill-<%= @order.payments.failed.count > 0 ? 'warning' : 'complete' %>">
-            <%= t(
-              'spree.payments_failed_count',
-              count: @order.payments.failed.count
-            ) %>
-          </span>
-        </td>
-      </tr>
-
-      <tr>
-        <td><strong><%= t('spree.avs_response') %>:</strong></td>
-        <td>
-          <span class="pill pill-<%= latest_payment.is_avs_risky? ? 'warning' : 'complete' %>">
-            <% if latest_payment.is_avs_risky? %>
-              <%= avs_response_code[latest_payment.avs_response] %>
-            <% else %>
-              <%= t('spree.success') %>
-            <% end %>
-          </span>
-        </td>
-      </tr>
-
-      <tr>
-        <td><strong><%= t('spree.cvv_response') %>:</strong></td>
-        <td>
-          <span class="pill pill-<%= latest_payment.is_cvv_risky? ? 'warning' : 'complete' %>">
-            <% if latest_payment.is_cvv_risky? %>
-              <%= cvv_response_code[latest_payment.cvv_response_code] %>
-            <% else %>
-              <%= t('spree.success') %>
-            <% end %>
-          </span>
-        </td>
-      </tr>
+    <tbody id="risk-analysis" data-hook="order_details_adjustments"  class="with-border">
+      <% order.payments.risky.each do |payment| %>
+        <tr class="">
+          <td><%= link_to payment.number, admin_order_payment_path(order, payment) %></td>
+          <td>
+            <span class="pill pill-warning"><%= t('spree.risky') %></span>
```

Comment:
Nit: At the moment, this column seems redundant, since it just shows "Risky" for every row. I'm guessing this column will show various degrees of riskiness in the future (Risky, Very Risky, ...)?

Reply:
Yes that's 100% true, in fact in the long run would make sense to turn risk assessment into a score with threshold, which is the most common solution outside and have different factors contribute to it (payments being just one part of this). 

But for now this is just a dent in that direction starting to move away from the old ways of 10 years ago 😊

Reply:
Good finding. Another UX/UI improvement (if we remove that redundant table column) could be moving the risky section in the right sidebar, close to some more global order information and not related to the currently open tab.

Reply:
@kennyadsl that would make sense, especially if paired with some order level way to convey risk information. 
You think it's fine to handle it separately or would you like to see it as part of this PR?

Reply:
Up to you @elia! Just let me know if you plan to do it separately and I'll proceed with merging this.

Reply:
@kennyadsl Yes! Please merge it as it is 🙏 
I would prefer to keep it separate and provide a good integration point for risk assessment and giving it more thought 💭